csrc/dynamic_transform.cpp

```diff
@@ -7,6 +7,7 @@
 // clang-format on
 #include <device_lower/utils.h>
 #include <dynamic_transform.h>
+#include <executor_kernel_arg.h>
```

Comment:
Why is this added?

Reply:
This file uses `KernelArgumentHolder` but didn't include its header. In the past, this was not failing because it included `evaluator_common.h` which then included `executor_kernel_arg.h`, but now, `evaluator_common.h` is no longer including `executor_kernel_arg.h`. In general, a file should include everything it uses. It should not rely on other files to include things for it.